apycula/gowin_bba.py

```diff
@@ -147,6 +147,17 @@ def write_timing(b, timing):
     b.u32(len(timing))
     b.ref(blk)
 
+def write_partnumber_packages(b, db):
+    import ipdb; ipdb.set_trace()
```

Comment:
uh breakpoints should go I think haha

Reply:
> Exactly as predicted! So what's up with the nextpnr side? I mean, isn't the chipdb format the same?

The structure has changed: a table with packages has been added.
You can put the next patch on nexpnr

https://github.com/yrabbit/nextpnr/commit/0e8a2999bda012175c1734abbf4efb76b515746c